### PR TITLE
Day data for pools

### PR DESCRIPTION
**Pull request by IainM22**
Daily snapshot of a Pool's state. Used for time series charts like 'Total staked over time'

### CODE REVIEW

File: schema.graphql [hunk 1]
```diff
@@ -176,4 +176,17 @@ type Funding @entity {
...
+}
+
+type PoolDayData @entity {
+  # pool id, day id
+  id: ID!
+
+  poolAddress: String!
+
+  date: Int!
+  totalStaked: BigDecimal!
```
Comment (devinaconley) [inline schema.graphql:188]: is it worth including total rewards and TVL in this pool snapshot as well?

File: schema.graphql [hunk 2]
```diff
@@ -176,4 +176,17 @@ type Funding @entity {
   originalAmount: BigDecimal!
   shares: BigDecimal!
   sharesPerSecond: BigDecimal!
-  }
+}
+
+type PoolDayData @entity {
```
Comment (devinaconley) [inline schema.graphql:181]: this schema is intended to be a snapshot or "point in time" of the pool stats right? not data specific to the day?

File: src/util/common.ts [hunk 3]
```diff
@@ -31,3 +31,24 @@ export function createNewPlatform(): Platform {
 
   return platform;
 }
+
+export function updatePoolDayData(geyser: Geyser, timestamp: number): PoolDayData {
```
Comment (devinaconley) [inline src/util/common.ts:35]: can we also hit this function from the periodic pricing update? that'll make sure we populate the snapshot stats every day

**Review by devinaconley — CHANGES_REQUESTED** [review]
Looking good! just left some minor comments

File: schema.graphql (hunk 2 above)
Comment (IainM22) [reply schema.graphql:181]: It is for everything except for volume, which is just the volume for that day

File: schema.graphql (hunk 1 above)
Comment (IainM22) [reply schema.graphql:188]: I have a chart which shows total rewards per day that is generated from funding data. I feel like that may be enough for now. TVL is probably worth adding though

File: src/util/common.ts (hunk 3 above)
Comment (IainM22) [reply src/util/common.ts:35]: good idea

File: schema.graphql (hunk 1 above)
Comment (devinaconley) [reply schema.graphql:188]: cool, makes sense :+1:

**Review by devinaconley — APPROVED** [review]
Looks great :+1:

**Pull request merged**